plugin/src/main/java/fr/mrmicky/worldeditselectionvisualizer/placeholders/PlaceholderManager.java

```diff
@@ -0,0 +1,39 @@
+package fr.mrmicky.worldeditselectionvisualizer.placeholders;
+
+import fr.mrmicky.worldeditselectionvisualizer.WorldEditSelectionVisualizer;
+import me.clip.placeholderapi.PlaceholderAPI;
+import me.clip.placeholderapi.expansion.PlaceholderExpansion;
+import org.bukkit.scheduler.BukkitRunnable;
+
+public class PlaceholderManager {
+    private final WorldEditSelectionVisualizer plugin;
+    private PlaceholderExpansion placeholders = null;
+
+    public PlaceholderManager(WorldEditSelectionVisualizer plugin) {
+        this.plugin = plugin;
+    }
+
+    public void hook() {
+        placeholders = new PlaceholderHandler(plugin);
+
+        new BukkitRunnable(){
+            @Override
+            public void run() {
+                placeholders.register();
+            }
+        }.runTask(plugin);
+    }
+
+    @SuppressWarnings("all")
+    public void unhook() {
+        if (placeholders != null) {
+            Integer version = Integer.parseInt(plugin.getServer().getPluginManager().getPlugin("PlaceholderAPI").getDescription().getVersion().split("\\.")[2]);
```

Comment:
[As pointed by LGTM](https://lgtm.com/projects/g/MrMicky-FR/WorldEditSelectionVisualizer/rev/pr-215e92bbda7d3c67c7630f7dd3d08ea51032209a) the `Integer` can be replaced by an `int`.

Also this only get the minor version, so if an old version of PlaceHolderAPI with a minor version higher than 5 is installed (like `2.8.6`) this would run `placeholders.unregister();` and cause a `NoSuchMethodError`